regexes/client/browsers.yml

```diff
@@ -387,7 +387,7 @@
 
 #Epiphany
 - regex: 'Epiphany(?:/(\d+[\.\d]+))?'
-  name: 'Epiphany'
+  name: 'GNOME Web'
```

Comment:
Please also update the comment above, and the name in the list of available browsers